src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -503,55 +507,52 @@ export class KeycloakUserService {
             return {
                 ok: false,
-                error: new KeycloakClientError('Could not update user status or custom attributes'),
+                error: new KeycloakClientError('Could not update user status or database'),
             };
         }
     }
 
-    private async updateCustomAttributes(
-        kcAdminClient: KeycloakAdminClient,
-        userId: string,
-        customAttributes: Record<string, string>,
-    ): Promise<void> {
-        const user: UserRepresentation | undefined = await kcAdminClient.users.findOne({ id: userId });
-        if (user) {
-            user.attributes = user.attributes ?? {};
-            for (const key in customAttributes) {
-                if (customAttributes.hasOwnProperty(key)) {
-                    user.attributes[key] = [customAttributes[key]];
-                }
-            }
-            await kcAdminClient.users.update({ id: userId }, user);
-        }
-    }
-
-    private async removeLockedAttributes(kcAdminClient: KeycloakAdminClient, userId: string): Promise<void> {
-        const user: UserRepresentation | undefined = await kcAdminClient.users.findOne({ id: userId });
-        if (user) {
-            user.attributes = user.attributes ?? {};
-            const filteredAttributes: Record<string, string[]> = Object.fromEntries(
-                Object.entries(user.attributes).filter(([key]: string[]) => !(key! in LockKeys)),
-            );
-
-            user.attributes = filteredAttributes;
-            await kcAdminClient.users.update({ id: userId }, user);
-        }
-    }
+    // private async updateCustomAttributes(
+    //     kcAdminClient: KeycloakAdminClient,
+    //     userId: string,
+    //     customAttributes: Record<string, string>,
+    // ): Promise<void> {
+    //     const user: UserRepresentation | undefined = await kcAdminClient.users.findOne({ id: userId });
+    //     if (user) {
+    //         user.attributes = user.attributes ?? {};
+    //         for (const key in customAttributes) {
+    //             if (customAttributes.hasOwnProperty(key)) {
+    //                 user.attributes[key] = [customAttributes[key]];
+    //             }
+    //         }
+    //         await kcAdminClient.users.update({ id: userId }, user);
+    //     }
+    // }
+
+    // private async removeLockedAttributes(kcAdminClient: KeycloakAdminClient, userId: string): Promise<void> {
+    //     const user: UserRepresentation | undefined = await kcAdminClient.users.findOne({ id: userId });
+    //     if (user) {
+    //         user.attributes = user.attributes ?? {};
+    //         const filteredAttributes: Record<string, string[]> = Object.fromEntries(
+    //             Object.entries(user.attributes).filter(([key]: string[]) => !(key! in LockKeys)),
+    //         );
+
+    //         user.attributes = filteredAttributes;
+    //         await kcAdminClient.users.update({ id: userId }, user);
+    //     }
+    // }
```

Comment:
If these are no longer needed, remove the methods altogether

Reply:
removed